src/System.Private.CoreLib/shared/System/Reflection/ConstructorInfo.cs

```diff
@@ -22,7 +22,6 @@ public abstract partial class ConstructorInfo : MethodBase
         public override bool Equals(object obj) => base.Equals(obj);
         public override int GetHashCode() => base.GetHashCode();
 
-        // Force inline as the true/false ternary takes it above ALWAYS_INLINE size even though the asm ends up smaller
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
@benaadams Did you intend to delete only the comment, not the associated attribute?

Reply:
Yeah, as I think its inaccurrate now (saying its actually a very small method that's just misunderstood by the Jit)